apis/v1beta1/shared_types.go

```diff
@@ -252,6 +253,11 @@ const (
 	// reconciled the route.
 	RouteReasonPending RouteConditionReason = "Pending"
 
+	//This reason is used with the "ResolvedRef" condition when one of
+	// route has not been accepted because a ParentRef references a Gateway
+	//in another namespace, but no ReferenceGrant allows such a reference.
+	RouteReasonParentRefNotPermitted RouteConditionReason = "ParentRefNotPermitted"
```

Comment:
```suggestion
	// This reason is used with the "ResolvedRef" condition when one of
	// route has not been accepted because a ParentRef references a Gateway
	// in another namespace, but no ReferenceGrant allows such a reference.
	RouteReasonParentRefNotPermitted RouteConditionReason = "ParentRefNotPermitted"
```